modules/acquia_cms_toolbar/css/acquia_cms_toolbar.css

```diff
@@ -1,671 +0,0 @@
-.acquia-cms-toolbar {
```

Comment:
Include style for below:
- Environment specific background color to environment variable.
- Admin menu styles.

Reply:
As per discussion in DSM included styles from Environment specific background color to environment variable but admin menu styles are removed.